ironfish-rust-nodejs/index.d.ts

```diff
@@ -0,0 +1,88 @@
+/// <reference types="node" />
```

Comment:
This should also be gitignored like the below -- I think maybe I forgot to update the gitignore file in my native Rust branch.